core-plugin/src/com/google/gct/idea/debugger/ui/CloudAttachDialog.java

```diff
@@ -486,7 +494,11 @@ public void run() {
                 public void run() {
                   DebugTarget targetSelection = null;
 
-                  if (debuggees != null && debuggees.getDebuggees() != null) {
+                  if (debuggees == null || debuggees.getDebuggees() == null || debuggees.getDebuggees().size() == 0) {
```

Comment:
getDebuggees().isEmpty() (more semantic and perhaps a tad faster when the list isn't empty)